src/vm/runners/cairo_runner.zig

```diff
@@ -174,6 +196,7 @@ pub const CairoRunner = struct {
         // self.program.deinit();
         self.function_call_stack.deinit();
         self.instructions.deinit();
+        self.vm.builtin_runners.deinit();
```

Comment:
CairoVM handles the builtin_runner management, so this would result in a double free

see:
https://github.com/keep-starknet-strange/cairo-zig/blob/main/src/vm/core.zig#L118